src/theme.js

```diff
@@ -82,23 +92,35 @@ const tints = [
   0.6,
   0.8
 ]
-const blues = tints.map(tint(blue))
-const grays = tints.map(tint(gray))
-const greens = tints.map(tint(green))
-const reds = tints.map(tint(red))
-const oranges = tints.map(tint(orange))
-const purples = tints.map(tint(purple))
+const blues = [lightBlue, blue, darkBlue]
+const grays = [lightGray, gray, darkGray]
+const greens = [lightGreen, green, darkGreen]
+const reds = [lightRed, red, darkRed]
+const oranges = [lightOrange, orange, darkOrange]
+const purples = [lightPurple, purple, darkPurple]
```

Comment:
As in the comment on the snapshot, I think these arrays should maybe have an extra duplicate color to  handle the case where someone might be using `purple3`, e.g. `[lightPurple, purple, darkPurple, purple]` then we can add a prop type warning, and remove the duplicate with a `1.0.0` release. This would ensure that the component still gets the color, just maybe not the 100% right one. Curious what others think though

Reply:
I don't know if anyone is using color3 yet. But since this is public, we should not take the risk of just removing that. I think that doing that would ensure backward compatibility for now. I will work on it.

Reply:
👌